src/app/home/home.component.css

```diff
@@ -0,0 +1,175 @@
+.box {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+   
+    
+  }
+  
+  .box div {
+      padding-left:15%;
+      padding-right:3%;
+      
+    width: 100%;
+    height: 100%;
+  }
+/* pogledati jos margine i ostaloo*/
+.search-bar{
+    margin-left: 16%;
+    padding-right: 5%;
+    margin-top:7%
+}
+
+
+.font-family{
+    font-family: 'Source Sans Pro', sans-serif;
+    font-weight: 600 !important;
+}
+/* for name ABH Restaurant */
+.font-size-20{
+    font-size: 20px;
+    font-weight: 600 !important;
+    
+}
+
+/* font 16px for nav-bar */
+.font-s-16{
+    padding: 0.5%;
+    font-size: 16px;
+ 
+}
+
+/* Header with backround image */
+header {
+    background: url(/assets/img/Bitmap.png) no-repeat center center;
+    width: 100%;
+    height: 100vh;
+    overflow: hidden;
+    background-size: cover;
+    -webkit-background-size: cover;
+    -moz-background-size: cover;
+    -o-background-size: cover;
+}
+
+/* Paddings for nav-bar */
+.header-navbar{
+  padding-top: 3%;
+  padding-left: 15%;
+  padding-right: 11%;
+  color:#FFFFFF;
+  font-weight: 400;
+}
+
+/*  Make gap beetwen Restaurant|Login  links */
+.m-r-1{
+    margin-right: 5%;
+}
+
+
+/*  Background image for carousel gallery */
+.middle-gallery{
+    background: url(/assets/img/background-specials.png) no-repeat center center;
+    width: 100%;
+    padding: 20px 10px 60px 10px;
+    height: 100vh;
+    overflow: hidden;
+    background-size: cover;
+    -webkit-background-size: cover;
+    -moz-background-size: cover;
+    -o-background-size: cover;
+}
+
+/* style for container, check out later!!!!! */
+.container{
+    position: relative;
+    size: 100%
+}
+
+.green-square {
+    background-color: #3D9970;
+    width: 100px;
+    height: 100px;
+    position: absolute;
+    left: 50%;
+    margin-left: -50px;
+  }
+
+.proba{
+    position: absolute;
+    left: 50%;
+    margin-left: -50px;
+}
+
+/* styles for center-text in header*/
+.header-text-center{
+    text-align: center;
+    padding-top:10% ;
+    color:#FFFFFF;
+    font-family: 'Source Sans Pro', sans-serif;
+}
+
+/* font size 48 */
+.font-size-48{
```

Comment:
I don't think it's a good idea to use classes like `.font-size-48`, instead add the necessary font properties to a specific class